.github/report-flaky-tests/index.js

```diff
@@ -88,35 +88,37 @@ const metaData = {
 				}
 			}
 
-			let testResultsList = body
-				.slice(
-					body.indexOf( TEST_RESULTS_LIST.open ) +
-						TEST_RESULTS_LIST.open.length,
-					body.indexOf( TEST_RESULTS_LIST.close )
-				)
-				/**
-				 * Split the text from:
-				 * ```
-				 * <!-- __TEST_RESULT__ --> Test result 1 <!-- /__TEST_RESULT__ -->
-				 * ...
-				 * <!-- __TEST_RESULT__ --> Test result 2 <!-- /__TEST_RESULT__ -->
-				 * <!-- __TEST_RESULT__ --> Test result 3 <!-- /__TEST_RESULT__ -->
-				 * ```
-				 *
-				 * into:
-				 * ```
-				 * [
-				 *   '<!-- __TEST_RESULT__ --> Test result 1 <!-- /__TEST_RESULT__ -->',
-				 *   '<!-- __TEST_RESULT__ --> Test result 2 <!-- /__TEST_RESULT__ -->',
-				 *   '<!-- __TEST_RESULT__ --> Test result 3 <!-- /__TEST_RESULT__ -->',
-				 * ]
-				 * ```
-				 */
-				.split(
-					new RegExp(
-						`(?<=${ TEST_RESULT.close })\n(?:\.\.\.\n)?(?=${ TEST_RESULT.open })`
-					)
-				);
+			let testResultsList = body.includes( TEST_RESULTS_LIST.open )
```

Comment:
Due to the previous error, some issues have been updated to an incorrect state without the open tag of the results list. We check here and discard the incorrect errors to start a new one.